src/ui/FieldTable/FieldValue.ts

```diff
@@ -216,6 +225,17 @@ export class FieldValue extends Component {
     return element;
   }
 
+  /**
+   * Render the caption with the text defined in the component options.<br/>
+   * Returns a <code>HTMLElement</code> containing the rendered caption as text.
+   */
+  public renderTextCaption(): HTMLElement {
```

Comment:
Does this really need to be public ?

If you think in term of an API, why would external code need to get an HTMLElement generated by the text caption.

This should be a private, and non-documented method.